<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventItemWidget</class>
 <widget class="QWidget" name="EventItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>941</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="eventNameLabel">
        <property name="text">
         <string>EventName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="placeLabel">
        <property name="maximumSize">
         <size>
          <width>99999</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="eventStartLabel">
        <property name="text">
         <string>EventStart</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="organizerLabel">
        <property name="maximumSize">
         <size>
          <width>99999</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Organizer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="addressLabel">
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="eventEndLabel">
        <property name="text">
         <string>EventEnd</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="eventTicketPrice">
        <property name="text">
         <string>EventTicketPrice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="eventCapacity">
        <property name="text">
         <string>Capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Buy ticket</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
